x/programs/runtime/import_program.go

```diff
@@ -49,18 +58,16 @@ func NewProgramModule(r *WasmRuntime) *ImportModule {
 
 				result, err := r.CallProgram(
 					context.Background(),
-					&newInfo)
-				if err != nil {
-					return nil, err
-				}
+					&newInfo,
+				)
 
 				// subtract the fuel used during this call from the calling program
 				remainingFuel := newInfo.RemainingFuel()
-				if err := callInfo.ConsumeFuel(parsedInput.Fuel - remainingFuel); err != nil {
-					return nil, err
```

Comment:
@dboehm-avalabs is it okay to treat this error the same as the `OutOfFuel` trap error that is in the runtime ?

Reply:
Realistically this should never actually error.  We check that there is enough fuel before calling this and we will fail much earlier if fuel isn't enabled.

Reply:
Thanks, so I will leave it that way